api/RTCStatsReport.json

```
@@ -255,9 +258,8606 @@
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "≤72"
+              },
+              "firefox_android": {
+                "version_added": "≤112"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "≤13.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "codecId": {
+          "__compat": {
+            "description": "<code>codecId</code> in 'outbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcrtpstreamstats-codecid",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "98"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "11"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "headerBytesSent": {
+          "__compat": {
+            "description": "<code>headerBytesSent</code> in 'outbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcoutboundrtpstreamstats-headerbytessent",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": "99> ≤101"
```

https://mdn-bcd-collector.gooborg.com/tests/api/RTCStatsReport/type_outbound-rtp/headerBytesSent
Test shown added (supported) in 100.
```suggestion
                "version_added": "100"
```